riomalos.py

```diff
@@ -0,0 +1,77 @@
+class KDramaList:
+    def __init__(self):
+        self.kdramas = []
+        self.watched_kdramas = []
+        self.max_kdramas = 100
```

Comment:
capitalize constant name - max_kdramas

Reply:
fix uncapitalize constant in commit ace84fb

Reply:
fix uncapitalize constant in line 8, fix in commit  8d1183b